### PR TITLE
Update nodes_and_scenes.rst

### PR DESCRIPTION
**Pull request by VictorDev200**
Changing "Character" to "Player"

### CODE REVIEW

**Comment by skyace65** [comment]
I think it would be better to just remove any mention of what the node is named, it's not necessary at all for that paragraph.

**Comment by VictorDev200** [comment]
@skyace65 I will remove the paragraph then

**Comment by skyace65** [comment]
You don't need to delete the whole paragraph, just `named "Character" or "Player"`

File: getting_started/step_by_step/nodes_and_scenes.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ The last characteristic is important. **Together, nodes form a tree**, which is
 feature to organize projects. Since different nodes have different functions,
 combining them produces more complex behavior. As we saw before, you can build a
 playable character the camera follows using a :ref:`CharacterBody2D <class_CharacterBody2D>`
-node named "Character", a :ref:`Sprite2D <class_Sprite2D>` node,
+node named "Character" or "Player", a :ref:`Sprite2D <class_Sprite2D>` node,
```
Comment (mhilbrunner) [inline getting_started/step_by_step/nodes_and_scenes.rst:34]:
```suggestion
node, a :ref:`Sprite2D <class_Sprite2D>` node,
```

**Comment by mhilbrunner** [comment]
Edited and merged. Thanks for contributing!

**Pull request merged**